static/styles/left-sidebar.css

```diff
@@ -82,7 +82,6 @@ ul.filters hr {
 li.active-filter,
 li.active-sub-filter {
     font-weight: 600 !important;
-    background: #ddedf6;
```

Comment:
Why does this also change the colors?  I thought we were just doing opacity changes...